### PR TITLE
Remove (wrong) GI_TYPELIB_PATH and XDG_DATA_DIRS

### PR DESCRIPTION
**Pull request by Flamefire**
Those are set to the correct folders in EasyBlock now

- [x] Requires https://github.com/easybuilders/easybuild-framework/pull/3133

Fixes https://github.com/easybuilders/easybuild-easyconfigs/issues/9520

### CODE REVIEW

**Comment by Flamefire** [comment]
@boegel What to do about the failures? Adding the python suffix will make other ECs fail to build

**Comment by boegel** [comment]
@Flamefire By means of exception we can comment out the `check_python_packages` call in `test/easyconfigs/easyconfigs.py` and then re-enable it in a follow-up PR...

**Comment by Flamefire** [comment]
Rebased and done by a less invasive change (single test removed only for the failing EC name)

File: test/easyconfigs/easyconfigs.py [hunk 1]
```diff
@@ -578,7 +578,8 @@ def check_python_packages(self, changed_ecs):
 
             # if Python is a dependency, that should be reflected in the versionsuffix
             # Tkinter is an exception, since its version always matches the Python version anyway
-            if any(dep['name'] == 'Python' for dep in ec['dependencies']) and ec.name != 'Tkinter':
+            if any(dep['name'] == 'Python' for dep in ec['dependencies']) and ec.name != 'Tkinter' and \
+                ec.name != 'GObject-Introspection':
```
Comment (boegel) [inline test/easyconfigs/easyconfigs.py]: @Flamefire Just a reminder: do open a PR to roll this back after #9528 is merged...

Comment (Flamefire) [reply test/easyconfigs/easyconfigs.py]: Please ping me again when this (finally) will be merged. Hooray for `sed`, this sanity checks  on the old ECs get annoying ;)

Comment (boegel) [reply test/easyconfigs/easyconfigs.py]: Thanks a lot for your efforts on this one! :+1:

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 126 out of 126 (126 easyconfigs in this PR)
node2412.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/243a7ebcef4fae4758500c328c3939da for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Flamefire!

File: test/easyconfigs/easyconfigs.py (hunk 1 above)
Comment (boegel) [reply test/easyconfigs/easyconfigs.py]: Fixed in #9577, thanks @Flamefire!

**Comment by Flamefire** [comment]
I rest my head in shame, how could that even succeed? I missed the colon in `https://` :(

**Pull request merged**